src/postprocess/ked.jl

```diff
@@ -0,0 +1,20 @@
+"""
+Computes the kinetic energy density (ked), 0.5 ∑ fn |∇ψn|^2.
+"""
+function compute_kinetic_energy_density(basis::PlaneWaveBasis, ψ, occupation)
+    @assert all(symop -> length(symop) == 1, basis.ksymops) == 1 
+    ked = zeros(eltype(basis), basis.fft_size)
+    for (ik, kpt) in enumerate(basis.kpoints)
+        for (n, ψnk) in enumerate(eachcol(ψ[ik]))
+            for α = 1:3
+                dαψnk = [im*(G+kpt.coordinate_cart)[α] for G in G_vectors_cart(kpt)] .* ψnk
+                dαψnk_real = G_to_r(basis, kpt, dαψnk)
+                ked .+= @. basis.kweights[ik] *
+                           occupation[ik][n] * 
+                           real( conj(dαψnk_real) * dαψnk_real)

```

Comment:
Does this actually work? * does matrix multiplication, you want .* here

Reply:
It does work and I think it's because of the `@.` after `ked .+ =`.  That's how the `current.jl` function did it (with `@.` so it didn't use `.*` to broadcast the multiplication). I could remove the ` @.` and use `.*` instead, do you have a preference?

Reply:
Oh of course you're right, sorry! No need to change anything then